cic/addon/pre-requisites/secure

```diff
@@ -8,6 +8,10 @@ teedata=`sudo parted -l | grep -i teedata`
 trusty_host=`dmesg | grep trusty`
 tos_img=`dmesg | grep trusty | grep 'error -22'`
 trusty_init=`dmesg | grep trusty | grep 'initializ'`
+tos_dev=`sudo blkid|grep "\"tos\"" | awk '{print $1}' | tr -d ':'`
```

Comment:
suggest to following same coding style, add space before and after '|' symbol:

tos_dev=`sudo blkid | grep "\"tos\"" | awk '{print $1}' | tr -d ':'`
vbmeta_dev=`sudo blkid | grep "\"vbmeta\"" | awk '{print $1}' | tr -d ':'`